qiskit_dynamics/models/__init__.py

```diff
@@ -107,7 +107,7 @@
 
 .. code-block:: python
 
-    model.evaluation_mode = 'sparse'
+    model.evaluation_mode = "sparse"
```

Comment:
Is there a reason to prefer double tick here?

Reply:
Just because it's the `black` convention.